### PR TITLE
change: rename to starname, remove unused packages

### PR DESCRIPTION
**Pull request by fdymylja**

### CODE REVIEW

**Comment by iov-bot-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/iov-one/weave/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by davepuchyr** [comment]
`curl -X POST "http://localhost:1317/starname/query/resolve" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"starname\":\"dave*iov\"}"`

yields

` "error": "unknown request: no custom querier found for route starname"`

@fdymylja, do you have to update `storeName` passed to https://github.com/iov-one/iovns/blob/refactor-domain/x/starname/client/rest/routes.go#L102?

File: x/starname/types/naming.go [hunk 1]
```diff
@@ -3,13 +3,9 @@ package types
...
 	// DomainStore key defines the store key used to store domains information
-	DomainStoreKey = "domain"
-	// AccountStoreKey defines the store key used to store account information
-	AccountStoreKey = "account"
-	// IndexStoreKey defines the store key used to store indexing information
-	IndexStoreKey = ModuleName + "index"
+	DomainStoreKey = "starname"
 	// RouterKey defines the path used to interact with the domain module
 	RouterKey    = ModuleName
 	QuerierAlias = "starname"
```
Comment (davepuchyr) [inline x/starname/types/naming.go:11]: Would making `QuerierAlias = "domain"` be expeditious?

Comment (fdymylja) [reply x/starname/types/naming.go:11]: I'm not understanding where htis is coming from... I'll check now...

Comment (fdymylja) [reply x/starname/types/naming.go:11]: > `curl -X POST "http://localhost:1317/starname/query/resolve" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"starname\":\"dave*iov\"}"`
> 
> yields
> 
> ` "error": "unknown request: no custom querier found for route starname"`
> 
> @fdymylja?

"custom" subpath is missing, full path is: custom/module-name/query-path

**Comment by davepuchyr** [comment]
> `curl -X POST "http://localhost:1317/starname/query/resolve" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"starname\":\"dave*iov\"}"`
> 
> yields
> 
> ` "error": "unknown request: no custom querier found for route starname"`
> 
> @fdymylja?

Similarly
```
[dave@debt build]$ ./iovnscli q starname resolve --starname 'dave*iov'
ERROR: unknown request: no custom querier found for route starname
```

(I used [genesis.json](https://gist.githubusercontent.com/davepuchyr/e1482e63cb81443cde1616f353c4779f/raw/d00565696b51a338de2d36210fbc671c946076e9/genesis.json), which includes [dave*iov](https://gist.github.com/davepuchyr/e1482e63cb81443cde1616f353c4779f#file-genesis-json-L6874).)

**Pull request merged**